integration/test/test_frequency_hint_usage.py

```diff
@@ -140,15 +140,15 @@ def setUpClass(cls):
             launcher.set_num_rank(num_rank)
             launcher.run('test_' + test_name)
 
-            # EE run
-            launcher = geopm_test_launcher.TestLauncher(app_conf,
-                                                        ee_agent_conf,
-                                                        cls._ee_report_path,
-                                                        cls._ee_trace_path,
-                                                        time_limit=time_limit)
-            launcher.set_num_node(num_node)
-            launcher.set_num_rank(num_rank)
-            launcher.run(test_name)
+            # EE run - Not available; Removed from Agent factory.
```

Comment:
On the fence:

One might argue that we should delete all tests in place for an agent since it was removed from the factory.  

On the other hand, we have left the EE agent code in the repository even if we have removed it from the factory.  The agent compiles, but does not execute without failure due to signals not provided by platformio.  Leaving the code in place but disabling the agent with the factory and disabling the tests for the agent with skip decorators is self consistent.  

By this logic we should leave the tests here in a disabled state until we (a) remove the EE agent source and then delete the tests or (b) re-implement the EE agent and modify the tests as needed once they are enabled again.

I don't love commented code, is the setUpClass still executed when the test is skipped?